### ISSUE TITLE
Autofocus focusable content after using skip links

### ISSUE THREAD

**Issue by tobi-or-not-tobi** [SAP/spartacus#6815]
Skip links have been introduced as part of the wider a11y initiative. While the skip links add the links and move the page, there's no focus being applied to the target element. This is causing some smaller issues, but more importantly it is not very convenient for the user, as another tab is required to start using the content. There are other improvements that we like to add:

- autofocus of the first focusable element (or the first "autofocus" element inside the target)
- add skip links to page sections, rather that page slots. The page slot approach has various complications. 
- simplify skip link configuration, harmonise 3 skip areas for all pages
- focus the element rather then using zoom-into-view. 
- leverage generic focus directives / features for the autofocus, tabfocus and escape focus
- move the focus back to the root when escape is used
- refocus the skip links if the escape key is used twice

**Comment by tobi-or-not-tobi** [comment]
QA:
- enable calydon if not standardized already
- use the tab to focus the skip links in the top of the window
- tab into any of those skip links 
- hit enter, and confirm that the first focusable element of the target is focused. 
- you can tab through the elements
- hit escape, confirm that the focus is dropped from the given focusable element
- hit escape once more, and confirm that the skip links reappear (even if we're scrolled down on the page)

**Comment by KateChuen** [comment]
Tested on dev10.
- verified that every time when there's no element in focus, and we press tab, the skip links appear.
- verified that we can tab into any of the 3 skip links
- verified that pressing enter puts the focus on an element in that category
- verified that we can tab through elements
- verified that pressing esc will remove the focus from the element
- verified that pressing esc when there's no focus will make the skip links appear again.

<img width="1031" alt="Screen Shot 2020-03-25 at 10 58 12 AM" src="https://user-images.githubusercontent.com/45409515/77551141-5e9a4e80-6e88-11ea-9a80-df881e26f328.png">
